src/components/MenuItem/MenuItem.tsx

```diff
@@ -8,12 +8,27 @@ import theme from "@root/theme";
 import { BodyText } from "../Typography";
 
 const StyledMenuItem = styled(MUIMenuItem)`
-	min-width: 150px;
 
 	&.MuiMenuItem-root {
 		min-height: 42px;
 	}
 
+	&.css-kk1bwy-MuiButtonBase-root-MuiMenuItem-root.Mui-selected {
+		background-color: ${theme.colors.white};
+		.menuLabel {
+			font-weight: 700;

```

Comment:
We already have a font-weight for this in our theme file. Let's try to prevent repeating our code as it will save us time in the future.